### DISCUSSION TITLE
Stream and snapshot url not showing in attributes

### DISCUSSION THREAD

**Discussion by rauwman** [tsightler/ring-mqtt#596] [General Discussion]
Hello, 
I’m trying to figure out the last 3 hours why the info sensor is not showing the stream and the snapshot url. I already reinstalled mqtt broker and your addon several times. I can add the devices without problems and can also see a snapshot where the sensors are. But when I click on the info sensor and the attributes I can only see the following lines: 
BatteryLevel
100
BatteryLife
100
FirmwareStatus
Up to Date
LastUpdate
March 27, 2023 at 10:05:02 PM
WirelessNetwork
Network_23
WirelessSignal
-69

It’s a Ring doorbelt gen 2. Without a protecting plan.
Any advice is appreciated.

**Comment by rauwman** [comment]
Update:
After rename my door belt and the chime to simple english names and readding the devices its working now.

**Reply by tsightler** [reply]
What kind of characters?
